recipes/libgeoip/build.bat

```diff
@@ -0,0 +1,6 @@
+mkdir build
+cd build
+
+nmake /f Makefile.vc
```

Comment:
My plan is to remove that environmental variable and set it in the `build.bat` to the `$PREFIX` is this ok? The docs say that the variable is the place to put the `GeoIP.dat` file and that this file should "be placed in the same place as GeoIP-enabled executable modules"